rasa/cli/data.py

```diff
@@ -303,16 +303,31 @@ def _convert_nlu_data(args: argparse.Namespace) -> None:
     from rasa.nlu.training_data.converters.nlu_markdown_to_yaml_converter import (
         NLUMarkdownToYamlConverter,
     )
+    from rasa.nlu.training_data.converters.watson_nlu_json_to_yaml_converter import (
+        WatsonTrainingDataConverter,
+    )
 
     if args.format in ["json", "md"]:
         rasa.nlu.convert.convert_training_data(
             args.data, args.out, args.format, args.language
         )
         telemetry.track_data_convert(args.format, "nlu")
     elif args.format == "yaml":
-        rasa.utils.common.run_in_loop(
-            _convert_to_yaml(args.out, args.data, NLUMarkdownToYamlConverter())
-        )
+        if isinstance(args.data, list):
+            data_path = args.data[0]
```

Comment:
why are we only taking the first file? I think we should take all of them, no?

Reply:
we could do something like `any(WatsonTrainingDataConverter().filter(Path(path)) for path in args.data)`

Reply:
oh. i was not aware that we can pass multiple path in args.data, i thought either you can provide a filepath or a directory